src/http/routes.cc

```diff
@@ -77,6 +77,9 @@ std::unique_ptr<http::reply> routes::exception_reply(std::exception_ptr eptr) {
             }
         }
         std::rethrow_exception(eptr);
+    } catch (const redirect_exception& _e) {
+       rep.reset(new http::reply());
+       rep->add_header("Location", _e.url).set_status(_e.status());
```

Comment:
Comment from previous PR cc @nyh.

> I think it's better to put this code in exception_reply(), which already handles all the other types of official exception, instead of splitting the exception-handling code between here and exception_reply()

https://github.com/scylladb/seastar/pull/1569#discussion_r1145927726